src/UriNormalize.php

```diff
@@ -16,6 +16,8 @@
 use function str_contains;
 
 /**
+ * @deprecated This filter will be removed in version 3.0 without replacement
```

Comment:
This text should be the same as the deprecation message in the documentation. (see above)

Reply:
Thanks for the review @froschdesign